### PR TITLE
OAK-9711 - Fix cold standby IT failures

### PR DESCRIPTION
**Pull request by kunal3112**

### CODE REVIEW

File: oak-segment-tar/src/test/java/org/apache/jackrabbit/oak/segment/standby/DataStoreTestBase.java [hunk 1]
```diff
@@ -172,20 +175,20 @@ public void testResilientSync() throws Exception {
...
+                        .build();
+                StandbyClientSync cl = StandbyClientSync.builder()
+                        .withHost(getServerHost())
+                        .withPort(serverPort.getPort())
+                        .withFileStore(secondary)
+                        .withSecureConnection(false)
+                        .withReadTimeoutMs(4_000)
+                        .withAutoClean(false)
+                        .withSpoolFolder(spoolFolder)
+                        .build()
```
Comment (reschke) [inline oak-segment-tar/src/test/java/org/apache/jackrabbit/oak/segment/standby/DataStoreTestBase.java:191]: could you please avoid white-space only changes? (unless this aligns us with 1.8...)

**Review by reschke — CHANGES_REQUESTED** [review]

File: oak-segment-tar/src/test/java/org/apache/jackrabbit/oak/segment/standby/DataStoreTestBase.java (hunk 1 above)
Comment (kunal3112) [reply oak-segment-tar/src/test/java/org/apache/jackrabbit/oak/segment/standby/DataStoreTestBase.java:191]: These changes align with 1.8

**Comment by kunal3112** [comment]
I am removing the testSyncBigBlob IT. The test was added to the 1.6 code when backporting changes from oak-1.8 (OAK-9451). Since ChunkedBlobStream is not supported in oak-1.6, it doesn't make sense to keep this test case around.

**Review by reschke — APPROVED** [review]

**Comment by kunal3112** [comment]
@reschke Will you merge the changes? I do not have write access.

**Comment by reschke** [comment]
will run tests once more - that said, I think this should be merged by somebody familiar with oak-segment - once it is merged I can do the release work.

**Comment by reschke** [comment]
Tests pass for me.

**Comment by ahanikel** [comment]
I'm fine with the changes.

**Pull request merged**